openstack/baremetalintrospection/v1/introspection/results.go

```diff
@@ -170,6 +170,7 @@ type Data struct {
 	MACs          []string                     `json:"macs"`
 	MemoryMB      int                          `json:"memory_mb"`
 	RootDisk      RootDiskType                 `json:"root_disk"`
+	Hostname      string                       `json:"hostname,omitempty"`
```

Comment:
nit: `omitempty` is only needed for structs that will send a JSON body in an API request. Since this is a result, it's not needed. I'll fix this post-merge.